website/docs/d/volume.html.md

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
You'll also want to update https://github.com/terraform-providers/terraform-provider-linode/blob/master/website/linode.erb

Reply:
Thanks Marques, I did this but can't find instructions on how to test the generated html file.

Reply:
It's this I believe https://github.com/terraform-providers/terraform-provider-linode/blob/master/GNUmakefile#L52